test/e2e/cypress/integration/profiles/verification/verify/errors.spec.ts

```diff
@@ -1,12 +1,12 @@
 import {
```

Comment:
This file looks like it has lots of changes, but I just had to indent the lines, to add the `for` loop to test both `link` and `code` with the same test cases. If you have an idea on how to improve that, LMK, otherwise, I think it's possible to review without whitespace changes.